.github/workflows/stale.yaml

```diff
@@ -17,7 +17,8 @@ jobs:
         with:
           stale-pr-message: This pull request is stale because it has been open 30 days with no activity. Remove stale label or comment or it will be closed in 7 days
```

Comment:
I think we should add a `stale-issue-message` as well.  a lot of the confusion here was that it was not at all clear that the bot was operating on this issue

Reply:
Should there be a `stale-issue-message` even if nothing happens after the label is added?

Reply:
I don't know, but in this case things happened on the issue (it was referenced in commits and PRs) and it was closed anyways.  so some paper trail seems warranted

Reply:
I guess that doesn't matter so much if we are no longer closing them however